### DISCUSSION TITLE
can't change conversation id

### DISCUSSION THREAD

**Discussion by mahi141414** [Soulter/hugging-chat-api#166] [Q&A] [answered]
hey , how can i change the conversation id?? in python script??!
i can't find any way,, like i want it to change its conversation id to be `65c79b98be888bd0e1ada0cC` which i got from `https://huggingface.co/chat/conversation/65c79b98be888bd0e1ada0c8`

**Comment by Soulter** [comment, marked as answer]
hello, you can use following codes to change conversation id
```
# Get conversation list(all conversations in your account)
conversation_list = chatbot.get_remote_conversations()
# Get conversation list(local)
conversation_list = chatbot.get_conversation_list()
```

update:
To change conversation id you can use 
```
chatbot.change_conversation(id)
```

**Reply by mahi141414** [reply]
thanksss 😭😭😭 i was not getting any solution for 3days! Thank you very much!! <3
